preprocess.sh

```diff
@@ -5,8 +5,8 @@
 [[ -z "$1" ]] && exit 1
 
 set +e
-rm -rf "$1"/versions/master/api_docs/python/tf/contrib/keras*
-rm -rf "$1"/versions/master/api_docs/python/tf/keras
+# rm -rf "$1"/versions/master/api_docs/python/tf/contrib/keras*
+# rm -rf "$1"/versions/master/api_docs/python/tf/keras
```

Comment:
Including Keras results in pages with duplicated names but different cases, which contradicts with dash's docset standard.

Reply:
I didn't run into any problems keeping `tf.keras` and `tf.contrib.keras`